bool_bot/meme.py

```diff
@@ -0,0 +1,121 @@
+import discord
+import asyncio
+from moviepy.editor import *
+import numpy as np
+import math
+
+import google_drive_feat
+import video
+import index
+
+# ====================================================
+# Video meme generator
+
+async def generate_meme_video(ctx, text, file_id, file_name):
+    video_name = google_drive_feat.download_file(file_id, file_name)
+
+    video_bytes = os.stat(google_drive_feat.temp_dir + video_name).st_size
+
+    if video_bytes > 8000000: # Over 8 MB
+        os.remove(google_drive_feat.temp_dir + video_name)
+        return await ctx.send("Size of original video exceed 8 MB. Discord cannot send it over 8 MB")
+
+
+    video = VideoFileClip(google_drive_feat.temp_dir + video_name).set_position("bottom")
+    video_width = video.size[0]
+    video_height = video.size[1]
+    time = video.duration
+
+    FONT_OFFSET = 0.08 # Determines fontsize based on width of video
+    fontsize = video_width * FONT_OFFSET
+    line_height = int(fontsize)
+
+    # Create text clip
+    num_of_lines = determine_lines(text, video_width, fontsize)
```

Comment:
if i understand correctly, num_of_lines is a list of strings. so maybe name it something more appropriate, since num_of_lines makes it seem like it is an integer or something like that.

Reply:
Yeah sounds good